README.md

```diff
@@ -30,25 +30,23 @@ To get started using `cozy-client` with (p)React, you need to create a `CozyClie
 import CozyClient from 'cozy-client'
 import CozyStackLink from 'cozy-stack-link'
 
-// We need to retrieve the domain name of the cozy instance and a token from the HTML
-// See https://cozy.github.io/cozy-docs-v3/en/dev/app/#behind-the-magic
-const root = document.querySelector('[role=application]')
-const data = root.dataset
-
 const client = new CozyClient({
   link: new CozyStackLink({
-    uri: `${window.location.protocol}//${data.cozyDomain}`,
-    token: data.cozyToken
+    uri: 'http://cozy.tools:8080',
+    token: '...'
   })
 })
 ```
+If you need guidance to get the URI of your instance and/or the token, see (https://cozy.github.io/cozy-docs-v3/en/dev/app/#behind-the-magic).
 
 #### Creating a provider
 
-To connect the client to your component tree, use a `CozyProvider` somewhere high in your app:
+All components that we want to connect to data need access to the client. We could pass it as a prop from component to component, but it'll quickly get tedious.
+We recommend that you use a `CozyProvider` somewhere high in your app. It will make the client available to all your components using the context:
```

Comment:
👍  for clarity. Maybe add a link on context ? `[context](https://reactjs.org/docs/context.html)`